src/public/modules/forms/base/directives/submit-form.directive.js

```diff
@@ -101,7 +102,25 @@ function submitFormDirective(
         if (isPersistentLogin) GTag.persistentLoginUse(scope.form)
 
         const query = $location.url().split('?')
-        const encodedQuery = query.length > 1 ? btoa(query[1]) : undefined
+        const queryString = query.length > 1 ? query[1] : undefined
```

Comment:
thanks! on further thought, decided not to use URL API, because there isn't a straightforward method in the api to return the full url ex query which is what we want. it seems more straightforward to simply split by `?`